src/features/settings/components/SettingTabContainer/index.tsx

```
@@ -0,0 +1,228 @@
+		data: pineconeIndexes,
+		refetch: refetchPineconeIndexes,
+		isFetching,
+		isError,
+		error,
+	} = useQuery({
+		queryKey: ["pinecone-indexes"],
+		queryFn: fetchPineconeIndexes,
+	});
+
+	useEffect(() => {
+		if (pineconeIndexes && pineconeIndexes.length > 0) {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: plugin.saveSettings

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)